packages/schematics/angular/service-worker/index.ts

```diff
@@ -189,7 +189,7 @@ function updateIndexFile(options: ServiceWorkerOptions): Rule {
 
     const indent = getIndent(closingHeadTagLine) + '  ';
     const itemsToAdd = [
-      '<link rel="manifest" href="./manifest.json">',
+      '<link rel="manifest" href="assets/manifest.json">',
```

Comment:
I believe the convention is to keep the manifest file in the root of the directory. Is there a reason we're deviating?